assets/model_monitoring/components/src/action_analyzer/action_analyzer_identify_problem_traffic/run.py

```diff
@@ -166,12 +166,12 @@ def assign_default_group(group_list, query, metrics, good_queries, bad_queries):
     """Assign default group for good and bad queries."""
     good_query_list = json.loads(good_queries)
     bad_query_list = json.loads(bad_queries)
-    if query in good_query_list:
-        good_group_name = f"{metrics}_good_group"
-        group_list = _append_value(group_list, good_group_name)
-    elif query in bad_query_list:
+    if query in bad_query_list:
```

Comment:
bad query is score <=3, good query is score ==5?

Reply:
yes